frontend/src/metabase/ui/components/buttons/Button/Button.styled.tsx

```diff
@@ -77,17 +82,29 @@ export const getButtonOverrides = (): MantineThemeOverride["components"] => ({
         };
       },
       filled: (theme, { color }: ButtonStylesParams) => {
-        const primaryColor = getPrimaryColor(theme, color);
-        const hoverColor = getHoverColor(theme, primaryColor);
+        const customPropertyColor = getCssCustomPropertyColor(
+          color as CustomColors,
+        );
+        const isThemeable = color === "brand";
+        const colors = isThemeable
+          ? {
+              default: "var(--mb-color-background-brand)",
+              hover:
+                "color-mix(in srgb, var(--mb-color-background-brand) 88%, transparent)",
```

Comment:
the hover color, in this case, is basically the same as before because `hoverColor` was based on `brand` color, and had the alpha value of `0.88`